cmake/dependencies.cmake

```diff
@@ -6,6 +6,9 @@ cmake_policy(SET CMP0054 NEW)
 
 ov_set_temp_directory(TEMP "${CMAKE_SOURCE_DIR}")
 
+# Create platform-specific subdirectory for cross-compilation support
+set(PLATFORM_SUBDIR "${CMAKE_SYSTEM_NAME}_${CMAKE_SYSTEM_PROCESSOR}")
```

Comment:
imho CMAKE_SYSTEM_PROCESSOR is enough,, we don't build linux binaries on windows)